migrations/1545154190802_create-users-table.js

```diff
@@ -34,6 +34,7 @@ CREATE UNIQUE INDEX index_users_on_email ON users USING btree (email);
 CREATE UNIQUE INDEX index_users_on_reset_password_token ON users USING btree (reset_password_token);
 `);
 
+  // do not override existing legacy ids in production (approximately 3000 users)
```

Comment:
we start
```suggestion
  // We start incrementing user id at 5000 because of a previous database migration that had ids from 1 to approx 3000. This avoids any conflict between old and new ids
```